product_supplierinfo_for_customer/migrations/12.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Eficent <http://www.eficent.com>
+# Copyright 2019 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+from psycopg2.extensions import AsIs
+
+
+def _move_model_in_data(env, ids, old_model, new_model):
+    renames = [
+        ('mail_message', 'model', 'res_id'),
+        ('mail_followers', 'res_model', 'res_id'),
+        ('ir_attachment', 'res_model', 'res_id'),
+        ('mail_activity', 'res_model', 'res_id'),
+        ('ir_model_data', 'model', 'res_id'),
+    ]
+    for rename in renames:
+        openupgrade.logged_query(
+            env.cr, """
+            UPDATE %s
+            SET %s = %s
+            WHERE %s IN %s AND %s = %s""",
+            (
+                AsIs(rename[0]),
```

Comment:
Use new `sql.SQL` syntax. It's much better than this. Or at least use a dictionary, as this is very hard to be understood with indexes.